test/expected/telemetry_community.out

```diff
@@ -46,6 +46,8 @@ SELECT
 FROM
   device_readings
 GROUP BY bucket;
+NOTICE:  refreshing continuous aggregate
+NOTICE:  continuous aggregate "device_summary" is already up-to-date
```

Comment:
Can we output either one or the other of these messages. It seems unnecessary to display both. Maybe move the refreshing message into refresh code and then call it via the internal refresh function with a boolean for optionally displaying the refreshing message?

Reply:
I moved the printout so that we get a message only if there is a range to refresh so now we get a single message which is either that it is refreshing, or that there were nothing to refresh.

Note that the call to `refresh_continuous_aggregate` will print the (old) "nothing to refresh" message but the "refreshing" message will only be generated when called from `CREATE MATERIALIZED VIEW`.